src/extras/helpers/ArrowHelper.js

```diff
@@ -94,6 +92,9 @@ THREE.ArrowHelper.prototype.setLength = function ( length, headLength, headWidth
 	if ( headLength < length ){
 		this.line.scale.set( 1, length - headLength, 1 );
```

Comment:
If you can confirm that scaling a dimension to 0 will result in a completely invisible object, then I am ok with this.